src/services/publish/publish.service.ts

```diff
@@ -11,19 +9,19 @@ const PLATFORM_SERVICES = {
 
 export class PublishService {
 
-    constructor(private eventsRepository: EventsRepository) { }
+    constructor() { }
```

Comment:
Just to note: this eventsRepository reference is also on develop branch. Maybe residue code from previous changes but doesnt seem like we need it for now